src/main/webapp/release_history.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
ditto, separate this into a different CL. Also, I thought eriyeza was working on these pages. Did you work on these too or are they in this PR because of some GIT merging issues?

Reply:
I don't know they are here too. I think they may have been added to the branch by mistake.

Reply:
Can you remove the changes to these files, so they don't show up in the PR.